src/main/java/com/jpn/lang/controller/EmployeeControllerApi.java

```diff
@@ -3,25 +3,55 @@
 import com.jpn.lang.model.request.EmployeePostRequest;
 import com.jpn.lang.model.request.EmployeePutRequest;
 import com.jpn.lang.model.response.EmployeeResponse;
-import org.springframework.http.ResponseEntity;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestParam;
-
 import java.util.ArrayList;
 
 public interface EmployeeControllerApi {
-    @PostMapping
+
+    @Operation(
+            tags = "ADD Employees",
+            description = "Add Employees",
+            responses = {
+                    @ApiResponse(
+                            description = "Success",
+                            responseCode = "200"
+                    ),
+                    @ApiResponse(
+                            description = "Data Not Found",
+                            responseCode = "404"
+                    )
+            }
+    )
+    @PostMapping(value = "/employees", consumes = "application/json", produces = "application/json")
```

Comment:
why did you add path in post mapping annotation here?
because now it has become 
http://localhost:8082/employees/employees

for post api

Reply:
i have deleted this line.